book/getting_started/system_requirements.md

```diff
@@ -0,0 +1,20 @@
+# System Requirements
+
+## Hardware
+- 64-bit architecture.
+- We recommend a minimum of 16GB RAM 
+- We recommend using an SSD or NVMe, as an HDD may cause the node to fall behind the chain tip. Note that SSD performance can deteriorate when close to capacity.
+
+
+### Archive node
+- For an Archive node of Ethereum Mainnet, we recommend at least 3TB storage space: 1.8TB for state and 200GB for temp files 
```

Comment:
No clue where we're gonna end up here so let's add a disclaimer at the top of the file to communicate that these are just ballparks based on existing node implementations, they're not representative, and we intend to benchmark and optimize for each audience